tiller_python_service/git_actions.py

```diff
@@ -148,6 +148,10 @@ def git_add_commit_push(repo_path, commit_message):
 
     return "Add, commit, and push operations were successful"
 
+def multiply_two_numbers(a, b):
+    print(a)
```

Comment:
We shouldn't be just using raw prints in this file, can we replace them with calls to the `logging` library instead? help tiller